mcstatus/utils.py

```diff
@@ -124,7 +124,7 @@ def deprecated(
 
 
 def deprecated(
-    obj: Any = None,  # noqa: ANN401
+    obj: Any = None,
```

Comment:
Just a quick note about this, as it might seem like a bug:

Ruff is able to pick up that this method continas typing overloads, which themselves don't use the `Any` type, and therefore the use of `Any` in the implementation here is allowed. It's a feature, not a bug.